index.dd

```diff
@@ -240,6 +240,30 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Subtyping with alias this,
+----
+struct Point
+{
+    private double[2] p;
+    // If Point.<function> isn't defined, use p.<function>
```

Comment:
I think the `Point.<function>` looks like some special syntax, and users students might be wondering what such syntax means.  I suggest, instead, saying "Forward all undefined symbols to `p`"

Reply:
Hehe I had something like this in the beginning, but then changed it to this, because I thought it's clearer to people -> reverting :)